homeworks/maksym.selenkov_MaksymSelenkov/9-animations/style.css

```diff
@@ -0,0 +1,379 @@
+html {
+  font-family: "Roboto", sans-serif;
+  scroll-behavior: smooth;
+}
+
+a {
```

Comment:
Nooo 😢
You should NEVER use tag selectors in CSS. Create a class and apply it to all elements. Review all your CSS code to replace such tag selectors